tests/testthat/test-util.R

```diff
@@ -60,43 +60,53 @@ test_that("new_report_id", {
 })
 
 test_that("secrets", {
-  skip_if_no_vault_server()
+  srv <- vaultr::vault_test_server()
```

Comment:
this block of 4 lines creates a new vault server that we will automatically throw away at the end of the test block